pom.xml

```diff
@@ -1,6 +1,122 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>com.mikhail.tarasevich</groupId>
-  <artifactId>CharCounter</artifactId>
-  <version>0.0.1-SNAPSHOT</version>
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.mikhail.tarasevich</groupId>
+	<artifactId>CharCounter</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+
+	<properties>
+		<junit-jupiter.version>5.7.0-M1</junit-jupiter.version>
+		<org.hamcrest.version>2.2</org.hamcrest.version>
+		<assertj-core.version>3.22.0</assertj-core.version>
+		<mockito-core.version>2.28.2</mockito-core.version>
+		<surefire-junit4.version>2.22.0</surefire-junit4.version>
+		<maven-surefire-plugin.version>2.22.0</maven-surefire-plugin.version>
+		<maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
+
+		<jacoco-maven-plugin.version>0.8.2</jacoco-maven-plugin.version>
+	</properties>
+
+	<dependencies>
+
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter</artifactId>
+			<version>${junit-jupiter.version}</version>
+			<scope>test</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>org.hamcrest</groupId>
+			<artifactId>hamcrest</artifactId>
+			<version>${org.hamcrest.version}</version>
+			<scope>test</scope>
```

Comment:
where you use hamcrest?

Reply:
I used it at the beginning, but then I forgot to remove it. removed.